### PR TITLE
Update lesson

### PR DESCRIPTION
**Pull request by jlboba**
Changes made:
- Configured for TS
- Added conclusion section + reminder to test; otherwise, biggest changes were again in the solution branch (which is, of course, available!)

### CODE REVIEW

File: README.md [hunk 1]
```diff
@@ -59,3 +63,17 @@ are captured in state.
 _When the form is submitted_, add a new task to the list with the text and
 category from the form. For the tests for this feature to pass, you'll need a
 callback prop named `onTaskFormSubmit` that takes a task object as an argument.
+
+## Conclusion
+
+Hooray! You've completed this mini project. This was no easy task, handing
```
Comment (lizbur10) [inline README.md]: Typo: should be "handling"

File: README.md [hunk 2]
```diff
@@ -59,3 +63,17 @@ are captured in state.
...
+state, events, and passing props are all difficult React concepts that can take
```
Comment (lizbur10) [inline README.md]: Suggested edit: "handling state and events and passing props are difficult..."

File: README.md [hunk 3]
```diff
@@ -59,3 +63,17 @@ are captured in state.
...
+some time wrapping one's head around. If you feel like you don't have a proper
+handle on them yet or found yourself referring back to previous examples, that's
+more than OK. In fact, that is expected at this stage. Deep understanding will
+come with time and practice.
+
+And still, you were able to complete a project that utilized all these
+fundamental React concepts - congratulations! Next, we'll build your React
+knowledge with more advanced topics, starting with something side effects and
```
Comment (lizbur10) [inline README.md]: "...something called side effects"?

**Review by lizbur10 — CHANGES_REQUESTED** [review]

File: README.md (hunk 1 above)
Comment (jlboba) [inline README.md]:
```suggestion
Hooray! You've completed this mini project. This was no easy task, handling
```

File: README.md (hunk 2 above)
Comment (jlboba) [inline README.md]:
```suggestion
state and events and passing props are difficult React concepts that can take
```

File: README.md (hunk 3 above)
Comment (jlboba) [inline README.md]:
```suggestion
knowledge with more advanced topics, starting with something called side effects and
```

**Pull request merged**